requirements.txt

```diff
@@ -5,6 +5,7 @@ python-magic~=0.4.27; sys_platform == 'linux' or sys_platform == 'darwin'
 python-magic-bin~=0.4.14; sys_platform == 'win32'
 python_json_logger~=2.0.4
 PyYAML~=6.0
+pydantic~=2.7.4
```

Comment:
this dependency seem to break connectors that use older versions of pydantic (1.10.17)